bundles/org.openhab.binding.ambientweather/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -46,6 +62,10 @@
 		<label>Feeling Temperature</label>
 		<description>Feeling temperature</description>
 		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
```

Comment:
Feeling temeperature is not a measurement, i think it is rather a calculation.
_A "feels like" temperature, also known as heat index or wind chill, is an adjusted temperature that indicates how hot or cold it actually feels to the human body, considering factors like humidity and wind speed alongside the actual air temperature. These calculations help people understand the potential impact of weather conditions on their comfort and safety._